core/src/main/java/com/linecorp/armeria/common/logging/BuiltInProperty.java

```diff
@@ -182,9 +309,82 @@
                         .collect(toImmutableList());
     }
 
+    static BuiltInProperty findByKey(String key) {
+        return keyToEnum.entrySet().stream()
+                        .filter(e -> e.getKey().equals(key))
+                        .map(Entry::getValue)
+                        .findFirst().orElse(null);
+    }
+
+    private static BaseEncoding lowerCasedBase16() {
+        return lowerCasedBase16;
+    }
```

Comment:
Probably not worth a method?

Reply:
I tried to avoid an illegal forward reference. Now, I found a workaround for this.
Let me use a qualified name. 😀

Reply:
Oops, the qualified name only works with a constructor.
https://stackoverflow.com/questions/30169153/illegal-forward-reference-in-enum/30169296

Reply:
Doh! No worries then. :sweat_smile: